### PR TITLE
Changes 04-07-2023 (v2.0)

### PR DESCRIPTION
**Pull request by ricowezenberg**
Changes 04-07-2023 (v2.0)

# Pull Request Template

By contributing, you agree that your contributions will be licensed under the projects original open source license.

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
Previous version code was outdated and not compatible with the latest ExO PowerShell module version. This version is build on the latest PowerShell module version and is also tested to be compatible with PowerShell 7.

Updated code to use the new 3.1.0 PowerShell modules 
Some arguments are moved to the command line instead of hardcoded in the script 
Updated authentication mechanism and credential storage location 
Added the option to run the script from a different server 
Cleaned up input fields and information in the script.

### Type of change

* [x] Bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [x] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

### Checklist (check all applicable):

* [x] My code follows the style guidelines of this project
* [x] I have performed a self-review of my own code
* [x] I have commented my code, particularly in _hard to understand_ areas
* [x] I have made corresponding changes to the documentation
* [x] My changes generate no new warnings
* [x] I have added tests that prove my fix is effective or that my feature works
* [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: VBO-IncludeSpecificRecipientTypes/README.md
Comment (carceneaux) [inline VBO-IncludeSpecificRecipientTypes/README.md:1]: Please update letting others know which Veeam Backup for Microsoft 365 versions the script was tested against and/or supports.

Here's a sample README you can use as a reference:

https://github.com/VeeamHub/powershell/blob/master/VB365-ExportMailboxToPST/README.md

File: .gitignore
Comment (carceneaux) [inline .gitignore:1]: Please revert your commit deleting this file as it's necessary for this project.

Comment (ricowezenberg) [reply .gitignore:1]: .gitignore file is not needed, as this was a ds store leftover. Is fixed now.

Comment (carceneaux) [reply .gitignore:1]: It is needed. 😄 It ensures folks submitting pull requests to this project don't accidentally include their AWS creds or `.DS_Store` files in their pull requests.

Here's a link to the file you deleted in your PR: https://github.com/VeeamHub/powershell/blob/master/.gitignore

Please add this file back and this PR should be ready to be merged.

Comment (ricowezenberg) [reply .gitignore:1]: Got it. Should be fixed now.

**Review by carceneaux — APPROVED** [review]
Looks great! Thanks for making those changes.

**Pull request merged**